index.js

```diff
@@ -4,7 +4,54 @@
  * Depending on the operation, either add up all of the numbers or subtract all of the numbers, from left to right.
  * @returns {number} The result of either adding all numbers or subtracting all numbers, depending on the arguments added to the command line.
  */
-function calculator() {}
+
+function calculator() {
+    const argument = process.argv.slice(2)
+    console.log(argument)
+    const numb = process.argv.slice(3)
+    console.log(numb)
+    let result = Number(numb[0])
+    if(typeof argument[0] !== "string"){
```

Comment:
The arguments returned by `process.argv` are always going to be a `string` so you might want to reconsider your conditional here or have some data type manipulation. ie:[ isNaN(argument[0])](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/isNaN).

If i run `node run.js 1 2 3`, I get `Invalid operation: 1`. How can you update this conditional to make sure you get the correct error message?